### ISSUE TITLE
replace mdl (ruby) with  JS mdl

### ISSUE THREAD

**Issue by brucellino** [mmul-it/kpa-marp-pandoc#1]
This image uses `mdl` to perform markdown linting, which brings a large dependency on the entire Ruby runtime. This greatly increases the size of the image.

We need the NodeJS runtime anyway for other tools - I suggest that we replace the ruby markdownlint with the NodeJS implementation: https://www.npmjs.com/package/markdownlint

**Comment by rascasoft** [comment]
Agreed. But don't forget all the Pandoc stuff, which is really huge.
I don't actually know how much we will save with npm js but it's a great idea!

**Comment by rascasoft** [comment]
In addition to this, let's consider the fact that we are using Ubuntu as base OS, but we might use Alpine (like the original Marp container does). This should save a considerable amount of space.

**Comment by brucellino** [comment]
Ok - let's keep this open and eat the elephant slowly :)
